### PR TITLE
added new lines

### PR DESCRIPTION
**Pull request by mdurakovic**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -7,6 +7,10 @@
     const addButton = document.querySelector('.addButton');
     const list = document.querySelector('ul');
 
+
```
Comment (mafilipo) [inline script.js:10]: remove blank space

Comment (Vrtlaroth) [inline script.js:10]: remove blank spaces

Comment (mafilipo) [reply script.js:10]: ti kopas :)

Comment (Vrtlaroth) [reply script.js:10]: zaboravija san kako se pise nepotreban ;)

File: script.js [hunk 2]
```diff
@@ -7,6 +7,10 @@
...
+
```
Comment (jsuton00) [inline script.js:11]: Remove empty lines

**Pull request merged**